### PR TITLE
Keep coding style when scoping files

### LINKED ISSUE #7: Keep coding style during prefixing process
Currently, the coding style is not kept when files are processed by the scoper. For exemple

```php
<?php

namespace MyNamespace;

use MyOtherNamespace\MyClass;

class MyClass
{
}
```

is modified like this

```php
<?php

namespace MyNamespace;

use MyOtherNamespace\MyClass;
class MyClass
{
}
```

### PR DESCRIPTION
**Pull request by matthieuauger**
Hi,

This PR fixes #7 

This code works well only because the MutableString is a reference and is shared between all Visitors. I'm not sure this is a good design, any feedback welcome.

Thanks

### CODE REVIEW

**Comment by matthieuauger** [comment]
@webmozart : Any feedback on this PR ?

Thanks

File: src/NodeVisitor/NamespaceScoperNodeVisitor.php [hunk 1]
```diff
@@ -6,23 +6,35 @@
...
     {
+        $this->content = $content;
         $this->prefix = $prefix;
     }
 
     public function enterNode(Node $node)
     {
         if ($node instanceof Namespace_ && null !== $node->name) {
-            $node->name = Name::concat($this->prefix, $node->name);
+            $prefix = (0 === strpos($node->name, '\\')) ? $this->prefix : $this->prefix.'\\';
```
Comment (webmozart) [inline src/NodeVisitor/NamespaceScoperNodeVisitor.php]: What exactly does this condition do?

**Comment by webmozart** [comment]
Hi @matthieuauger, thanks for adding this :) looks good! 

Can you add a tests for the individual classes?

**Comment by webmozart** [comment]
@matthieuauger any news?

**Comment by webmozart** [comment]
ping?

**Comment by webmozart** [comment]
ping @matthieuauger

**Comment by afurculita** [comment]
@matthieuauger any news on this?

**Comment by padraic** [comment]
@afurculita Working on an update to this in a personal branch which I can PR when ready. Seems mostly tests and a few small bits and pieces (e.g. licence/attribute for new class(es)).

**Comment by theofidry** [comment]
@padraic and I don't think this is worth pursuing: it adds non significant complexity for little value. We don't think the consumers will care of the CS of the prefixed code.

**Pull request closed without merging**